templates/CRM/Admin/Page/Reminders.tpl

```diff
@@ -36,13 +35,10 @@
           <td class="crm-scheduleReminders-is_repeat">{if $row.is_repeat eq 1}{ts}Yes{/ts}{else}{ts}No{/ts}{/if}&nbsp;</td>
           <td id="row_{$row.id}_status" class="crm-scheduleReminders-is_active">{if $row.is_active eq 1} {ts}Yes{/ts} {else} {ts}No{/ts} {/if}</td>
           <td>{$row.action|smarty:nodefaults|replace:'xx':$row.id}</td>
-          <td class="hiddenElement"></td>
         </tr>
       {/foreach}
     {else}
       <tr><td colspan="8">{ts}No Scheduled Reminders have been created.{/ts}</td></tr>
```

Comment:
This colspan probably should be 7 now?

Reply:
@vingle as you can see I have also removed one header at line 25 and so I removed one td at line 39. to keep the th:td ratio same. So I guess there is no need for colspan or are you seeing any breakage in local ?

Reply:
Not seeing any breakage - it just looked like you had reduced 8 `<td>`s in the table head and body to 7, so the <td> with a colspan of 8 should now span 7. But the browser should be able to handle it either way I think (ie if you set colspan 20 it would still only be 7 columns wide.)

Reply:
Ahh right, my bad its in the else part at line 41 that I overlooked.  I have changed colspan to 7. Can you please check the updated patch?